server/src/main/java/com/paypal/selion/proxy/SeLionSauceProxy.java

```diff
@@ -97,14 +268,59 @@ public URL getRemoteHost() {
         return remoteHost;
     }
 
-    public void teardown() {
-        super.teardown();
+    @Override
+    public void beforeSession(TestSession session) {
+        // nothing to do before a session
+    }
+
+    @Override
+    public void afterSession(TestSession session) {
+        // nothing to do after session
+    }
+
+    @Override
+    public void startPolling() {
+        // polls but does not disconnect the node, once down polling limit is reached
+        pollingThread = new Thread(new Runnable() { // Thread safety reviewed
+            int failedPollingTries = 0;
+            long downSince = 0;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using redundant field initializer for 'downSince'](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2414240556/issues/source?bid=2928425&fileBranchId=3233060#l286)